modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -16,6 +16,109 @@
+
+* Content Security Policy: +
+When using web office document servers or web apps like provided in our docker compose deployment examples, a CSP has been implemented to secure the environment. For details on web office/apps see the next list item. 
+
+* Fully customisable deployment examples ready for productive use including web office, based on docker compose. See the
+xref:next@ocis:ROOT:depl-examples/ubuntu-compose/ubuntu-compose-prod.adoc[Local Production Setup] and xref:next@ocis:ROOT:depl-examples/ubuntu-compose/ubuntu-compose-hetzner.adoc[Deployment on Hetzner]. Note that with these examples, you can easily enable additional web apps like `draw-io`, `external-sites`, `json-viewer`, `unzip`, `progressbars` and others. Though provided via the compose example setup, no support can be given for these web apps.
+--
+
+=== Known Issues
+
+The following is a list of known issues identified in Infinite Scale rolling releases 6.x that still exist:
+
+OCM::
++
+--
+* xref:6-6-0-known-issues[Known Issues in 6.6.0 (OCM)] and
+* xref:6-4-0-known-issues[Known Issues in 6.4.0 (OCM)]
+--
+
+NOTE: OCM is still in experimental phase!
+
+=== Deprecations
+
+* Deprecations announced in release 5.0.0 in section xref:5-0-0-deprecation-announcements[Deprecation Announcements] have finally been removed.
+
+=== Migrations
+
+See the xref:next@ocis:ROOT:migration/upgrading-ocis.adoc[Upgrading Infinite Scale] documentation for migration and upgrade steps.
+
+=== Breaking Changes
+
+Infinite Scale::
++
+--
+* The xref:next@ocis:ROOT:{s-path}/activitylog.adoc[activitylog] service has been added. +
+To make it work, the Infinite Scale config must be updated. See the link in the xref:migrations[migrations] section above for details how to do so.
+* The xref:next@ocis:ROOT:{s-path}/collaboration.adoc[collaboration] service has been added. +
+If you have used web office document servers with the external https://github.com/cs3org/wopiserver[cs3org/wopi] server, you should switch to the collaboration service. See the link in the xref:migrations[migrations] section above for details how to do so.
+--
+
+Web UI::
++
+--
+* Deprecated config options have finally been removed, see https://github.com/owncloud/web/pull/11256[#11256]. +
+Note that these config options were able to be set via the `WEB_UI_CONFIG_FILE` environment variable. Having one of the deprecated set, will not have any impact.
+* The `draw-io` app has been removed as default Web app. It's now part of the https://github.com/owncloud/web-extensions[web-extensions repository].
+* The `importer` app has been removed as default Web app. It's now part of the https://github.com/owncloud/web-extensions[web-extensions repository].
```

Comment:
```suggestion
* The `draw-io` app has been removed as a default Web app. It's now part of the https://github.com/owncloud/web-extensions[web-extensions repository].
* The `importer` app has been removed as a default Web app. It's now part of the https://github.com/owncloud/web-extensions[web-extensions repository].
```

I suppose that there are multiple apps in the "list of default web apps", and these 2 apps have been removed from the list. Is that what is meant?